### ISSUE TITLE
About MSVD

### ISSUE THREAD

**Issue by sharontaozi** [albanie/collaborative-experts#5]
In dataset MSVD, the number of text descriptions to each video is different. How do you deal with this problem in your experiment?

**Comment by albanie** [comment]
During training, we sample one description from the collection for each forward pass.

**Comment by albanie** [comment]
@sharontaozi, closing for now (but feel free to re-open if it's still unclear).

**Comment by xixiareone** [comment]
The article mentions that "where they randomly chose 5 ground-truth sentences per video. We use the same setting when we compare with that approach".Does the training set, validation set and test set all take 5 sentences at random?
Not all sentences are used in training set, validation set and test set?
